src-ts/tools/work/work-list-title-renderer/WorkListTitleRenderer.tsx

```diff
@@ -0,0 +1,41 @@
+import { FC, SVGProps } from 'react'
+
+// TODO: export these from the svg barrel file
+import { ReactComponent as IconWorkTypeDataExploration } from '../../../lib/svgs/work-type-data-exploration.svg'
```

Comment:
Man, there's got to be a way to tame these ugly relative paths. Someday, but not today...

Reply:
once the TODO is done, it would be `from '../../../lib'`, which I don't mind. 

I actually like people to see exactly the relationship the current file has to the item it's importing. I think it's a flag to reduce nesting when it gets too long.

... but that's just my opinion.